src/service.js

```diff
@@ -39,8 +40,11 @@ let ServiceHolderDef;
  * @return {!Object} The service.
  */
 export function getExistingServiceForWindow(win, id) {
-  return dev().assert(
-      win.services[id] && win.services[id].obj, `${id} service not found.`);
+  const exists = win.services && win.services[id] && win.services[id].obj;
+  if ((getMode().localDev || getMode().test) && !exists) {
```

Comment:
disregard. removed this and fixed actual assertion to check every access to get a better error message with stack trace